src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
+using NuGet.Commands;
+using NuGet.Common;
+using static NuGet.Commands.VerifyArgs;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal partial class VerifyCommand
+    {
+        internal static void Register(CommandLineApplication app,
+                              Func<ILogger> getLogger,
+                              Action<LogLevel> setLogLevel,
+                              Func<IVerifyCommandRunner> getCommandRunner)
+        {
+            app.Command("verify", verifyCmd =>
+            {
+                CommandArgument packagesPath = verifyCmd.Argument(
+                    "<packages-path>",
+                    Strings.VerifyCommandPackagePathDescription,
+                    multipleValues: true);
+
+                CommandOption all = verifyCmd.Option(
+                    "--all",
+                    Strings.VerifyCommandAllDescription,
+                    CommandOptionType.NoValue);
+
+                CommandOption fingerPrint = verifyCmd.Option(
+                    "--certificate-fingerprint",
+                    Strings.VerifyCommandCertificateFingerprintDescription,
+                    CommandOptionType.MultipleValue);
+
+                CommandOption verbosity = verifyCmd.Option(
+                    "-v|--verbosity",
+                    Strings.Verbosity_Description,
+                    CommandOptionType.SingleValue);
+
+                verifyCmd.HelpOption("-h|--help");
+                verifyCmd.Description = Strings.VerifyCommandDescription;
+
+                verifyCmd.OnExecute(async () =>
+                {
+                    ValidatePackagesPath(packagesPath);
+
+                    VerifyArgs args = new VerifyArgs();
+#if IS_DESKTOP
+                    args.PackagePath = packagesPath.Value;
+#else
+                    args.PackagesPath = packagesPath.Values;
+#endif
```

Comment:
Why do you need to special-case this? Also, isn't XPlat always `IS_DESKTOP = false`?

Reply:
[NuGet.CommandLine.XPlat.csproj](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj#L7) has `Target Frameworks` property set to `TargetFrameworksExeForSigning`. As per [props](https://github.com/NuGet/NuGet.Client/blob/dev/build/common.project.props#L23) file this project, targets `net472`, `netcoreapp2.1` and `netcoreapp5.0` frameworks that's why I thought of adding this conditional compilation symbol. I will check if removing this would work.

Reply:
I think we do ship the net472 NuGet.CommandLine.XPlat. The .NET (Core) SDK in VS uses it.

Reply:
resolving this conversation because the compilation symbol is now removed